val-ansible-role/valkey/tasks/configure.yml

```diff
@@ -0,0 +1,51 @@
+- name: Create configuration directory
```

Comment:
task name should be like
configure | Create configuration directory
follow below for details
https://ansible.readthedocs.io/projects/lint/rules/name/#nameprefix